src/main/no/nav/pto/veilarbfilter/routes/MineLagredeFilterRoutes.kt

```diff
@@ -32,27 +32,36 @@ fun Route.conditionalAuthenticate(useAuthentication: Boolean, build: Route.() ->
     return route
 }
 
-fun Route.mineLagredeFilterRoutes(mineLagredeFilterService: FilterService, useAuthentication: Boolean) {
+fun Route.mineLagredeFilterRoutes(mineLagredeFilterService: MineLagredeFilterServiceImpl, useAuthentication: Boolean) {
     conditionalAuthenticate(useAuthentication) {
-        route("/minelagredefilter") {
+        route("/minelagredefilter/{enhetId}/") {
```

Comment:
This means that it is important to also update all applications that uses veilarbfilter?

Reply:
only our frontend application is using mine filter for now, so its safe to change little bit endpoind. I also updated frontend, so we will need to releaes both at similar time (probably outside working hours)